### PR TITLE
[Clp] Adr doc

### PR DESCRIPTION
**Pull request by Jedi2002**
First draft for the ADR . 
Looking for recommendations for things to include .

### CODE REVIEW

**Comment by AustinoBombino** [comment]
One thing I can see to include is the corrected pool units formula

**Comment by Jedi2002** [comment]
> One thing I can see to include is the corrected pool units formula

Yeah m This document was created before we decided to change the formula. Ill let thsi PR be ,and Finalize the document when when done with the rest of the tasks

**Comment by Jedi2002** [comment]
> One thing I can see to include is the corrected pool units formula

Yeah , This document was created before we made the changes , updated the formula now

**Comment by AustinoBombino** [comment]
cool, looks good

**Review by AustinoBombino — APPROVED** [review]

File: docs/clp-adr.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+### Basic Terminology 
+-Asset : An asset is most basic unit of a CLP . It Contains source chain, symbol and ticker to identify a token .
+```golang
+SourceChain: ETHEREUM
+Symbol: ETH
+Ticker: ceth
+
+SourceChain: SIFCHAIN
+Symbol: RWN
+Ticker: rwn
```
Comment (utx0) [inline docs/clp-adr.md:33]: need to update this too `rowan` to match smart-contact land.

**Review by utx0 — CHANGES_REQUESTED** [review]
Minor change.

File: docs/clp-adr.md (hunk 1 above)
Comment (Jedi2002) [reply docs/clp-adr.md:33]: both the ticker and the symbol ?

Comment (utx0) [reply docs/clp-adr.md:33]: @Jedi2002 please open up an issue for us to disucss the ticker vs symbol conversation and please set the context for us as this is a little unclear for me right now. Then we can close the loop on this ADR and update the code as needed too. Thanks

**Review by utx0 — APPROVED** [review]

**Pull request merged**